api/types/okta.go

```diff
@@ -356,7 +356,7 @@ func (o *OktaAssignmentActionV1) SetStatus(status string) error {
 		}
 	case OktaAssignmentActionV1_FAILED:
 		switch status {
-		case constants.OktaAssignmentActionStatusProcessing:
+		case constants.OktaAssignmentActionStatusPending:
```

Comment:
I'd also add a `default` on the outer switch, so that we don't risk letting some state transition slip by.